src/main/java/io/snyk/snyk_maven_plugin/download/ExecutableDownloader.java

```diff
@@ -63,11 +67,28 @@ public static File ensure(URL cliDownloadURL, File cliFile, String updatePolicy,
         }
     }
 
+    static String downloadToString(URL url) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(url.openStream(), StandardCharsets.UTF_8));
+        String all = br.lines().collect(Collectors.joining("\n"));
+        return all;
+    }
+
+    public static String getLatestVersion() throws IOException {
+        String version = "v" + downloadToString(new URL("https://static.snyk.io/cli/latest/version")).trim();
```

Comment:
Unrelated to this PR, but it makes me uneasy that every integration needs to add that damn "v" to the URL...

Reply:
Also a nit I'm unsure about: would it make sense to use `String.format` here as well?

```java
return new URL(String.format(SNYK_CLI_DOWNLOAD_FORMAT, "latest", "version"));
```

Reply:
I think that version endpoint should've returned the full path. Something like "https://static.snyk.io/cli/v1.532.0". Avoids hardcoding URLs into every integration and that "v" issue.